tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/BaseLinker.scala

```diff
@@ -93,8 +82,8 @@ final class Linker(semantics: Semantics, outputMode: OutputMode,
       }
     }
 
-    val analysis = logTime(logger, "Linker: Compute reachability") {
-      val analyzer = new Analyzer(semantics, outputMode, reachOptimizerSymbols,
+    val analysis = logger.time("Linker: Compute reachability") {
+      val analyzer = new Analyzer(semantics, symbolRequirements,
           allowAddingSyntheticMethods = true)
       analyzer.computeReachability(infoInput)
```

Comment:
Maybe take this opportunity to privatize the `class Analyzer`, and simply expose a `def computeReachability(...): Analysis` in the `Analyzer` companion object.